openassessment/templates/openassessmentblock/oa_base.html

```diff
@@ -3,7 +3,7 @@
 <div class="wrapper wrapper--xblock wrapper--openassessment theme--basic">
     <div class="openassessment problem" id="openassessment">
         <div class="wrapper--grid">
-            <h2 class="openassessment__title problem__header">{{ title }}</h2>
+            <h2 class="openassessment__title problem__header">{% trans title %}</h2>
```

Comment:
I see now.  I think you should then have `_("Open Response Assessment")` in the Python file, or the string may not be collected for translation at all.  You need to somehow ensure that the string is lexically marked as translatable (so that it will be collected), and also ensure that at run time the string is translated.  Often a simple `_()` function call is enough, but in your case it is trickier.